source/changelogs/core-2024.6.markdown

```diff
@@ -0,0 +1,13 @@
+---
+title: Full changelog for Home Assistant Core 2024.6
+description: Detailed changelog for the Home Assistant Core 2024.6 release
+replace_regex: \s\(\[?[a-z0-9\-\s_]+\]?\)$
+---
+
+These are all the changes included in the Home Assistant Core 2024.6 release.
+
+For a summary in a more readable format:
+[Release notes blog for this release](/blog/2024/06/05/release-20246/).
+
+## All changes
+
```

Comment:
quiet empty here :monocle_face: :see_no_evil:

Reply:
Yes will add them in a minute, just cut the release.